documentation/docs/examples/pointcloud.mdx

```diff
@@ -0,0 +1,50 @@
+---
+id: pointcloud
+title: Pointcloud models
+description: Usage of pointcloud models with Cognite3DViewer
+---
+
+`Cognite3DViewer` can be used to load photogrammetry models from Cognite Data Fusion.
+It works in exactly the same way as for [CAD](./cad-basic.mdx).
+
+<!--
+Notice however that pointcloud models are different.
+    TODO: here we should have some more meaningfull explanation about coloring/traversing etc
```

Comment:
@larsmoa can I ask you to write the doc itself here? Of course, I can try to do it but I guess I'll spend much time on that, and eventually you'll be editing it anyway 😄 3d concepts isn't something I can easily write about.